proc/stack.go

```diff
@@ -14,9 +14,16 @@ func (nra NoReturnAddr) Error() string {
 }
 
 type Stackframe struct {
-	Location
-	CFA int64
-	Ret uint64
+	// Address the function above this one on the call stack will return to
+	Return Location
```

Comment:
I think Return is overloaded here. How about something like `CurrentLocation`, basically something that conveys "this is the location we will be at once we are back in this frame". I understand the motivation for `Return` since it's the location that the function invoked from this frame will return to, however just reading through the source code you could also think it means that it is the return address of this function, not the one above it. Does that make sense?